Examples/CoulombSum3d/CoulombSum3d.cpp

```diff
@@ -225,6 +225,8 @@ int main(int argc, char** argv)
         tuner.SetValidationMethod(ktt::ValidationMethod::SideBySideComparison, 0.01);
     }
 
+    tuner.SetProfileBasedSearcher(kernel, "../../ProfileSearcher/models/1070-coulomb_output_DT.sav");
```

Comment:
Do we want to use profile searcher in this example? It is already used in the tutorial 08. If so, perhaps add the path to ML data to the beginning, where kernel source file paths are located as well (with correct Windows/Linux directories)